embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/SchemaType.kt

```diff
@@ -1,9 +1,20 @@
 package io.embrace.android.embracesdk.arch.schema
 
+import io.embrace.android.embracesdk.arch.schema.SchemaKeys.AEI_RECORD
+import io.embrace.android.embracesdk.arch.schema.SchemaKeys.CUSTOM_BREADCRUMB
+import io.embrace.android.embracesdk.arch.schema.SchemaKeys.LOG
+import io.embrace.android.embracesdk.arch.schema.SchemaKeys.VIEW_BREADCRUMB
 import io.embrace.android.embracesdk.internal.logs.EmbraceLogAttributes
 import io.embrace.android.embracesdk.internal.utils.toNonNullMap
 import io.embrace.android.embracesdk.payload.AppExitInfoData
 
+internal object SchemaKeys {
+    internal const val CUSTOM_BREADCRUMB = "custom-breadcrumb"
+    internal const val VIEW_BREADCRUMB = "view-breadcrumb"
+    internal const val AEI_RECORD = "aei-record"
+    internal const val LOG = "emb-log"
+}
+
 internal sealed class SchemaType(
     val telemetryType: TelemetryType,
     val name: String,
```

Comment:
nit: maybe we can do this more restrictive and turn it to be of type `SchemaKey` and `SchemaKey` be an enum. 
But it's just a detail... it's ok if you prefer it this way.

Reply:
I will keep this as-is for now since `SchemaType` is already a sealed class, but we can maybe reassess if any other functionality ever gets added to the string constants?